lib/DiscoveryStreamFeed.jsm

```diff
@@ -1072,10 +1072,11 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
   }
 };
 
+// Hardcoded version of layout_variant `3-col-7-row-octr`
 defaultLayoutResp = {
   "spocs": {
-    "url": "https://getpocket.cdn.mozilla.net/v3/firefox/unique-spocs",
-    "spocs_per_domain": 1,
+    "url": "https://getpocket.cdn.mozilla.net/v3/firefox/unique-spocs?consumer_key=$apiKey",
```

Comment:
Did this API change on the server to reject `consumer_key`-less requests? Any users already seeing discovery stream will be getting the `400 bad request` response -- expected?

Reply:
Actually, looks like spocs are currently requested via consumer_key as part of POST body. What is the server actually wanting? Both consumer_key in the url params and the post body?

Reply:
@mmiermans Can you answer this Q?

Reply:
It hasn't changed; `consumer_key` has always been required. Without it, the response will be `400 Bad Request` and the `X-Error` header will say `Missing API key. Get an API key at http:\/\/getpocket.com\/api\/.`.

Good thing to call our in review, and thanks for fixing it @gvn!